components/account/AccountMoreButton.vue

```diff
@@ -121,75 +156,82 @@ async function removeUserNote() {
             :command="command"
-            @click="toggleMuteAccount (relationship!, account)"
+            @click="toggleMuteAccount (relationship!, account, 'profile.more.mute')"
           />
           <CommonDropdownItem
+            is="button"
             v-else
             :text="$t('menu.unmute_account', [`@${account.acct}`])"
             icon="i-ri:volume-up-fill"
             :command="command"
-            @click="toggleMuteAccount (relationship!, account)"
+            @click="toggleMuteAccount (relationship!, account, 'profile.more.unmute')"
           />
 
           <CommonDropdownItem
+            is="button"
             v-if="!relationship?.blocking"
             :text="$t('menu.block_account', [`@${account.acct}`])"
             icon="i-ri:forbid-2-line"
             :command="command"
-            @click="toggleBlockAccount (relationship!, account)"
+            @click="toggleBlockAccount (relationship!, account, 'profile.more.block')"
           />
           <CommonDropdownItem
+            is="button"
             v-else
             :text="$t('menu.unblock_account', [`@${account.acct}`])"
             icon="i-ri:checkbox-circle-line"
             :command="command"
-            @click="toggleBlockAccount (relationship!, account)"
+            @click="toggleBlockAccount (relationship!, account, 'profile.more.unblock')"
           />
 
           <template v-if="getServerName(account) !== currentServer">
             <CommonDropdownItem
+              is="button"
               v-if="!relationship?.domainBlocking"
               :text="$t('menu.block_domain', [getServerName(account)])"
               icon="i-ri:shut-down-line"
               :command="command"
-              @click="toggleBlockDomain(relationship!, account)"
+              @click="toggleBlockDomain(relationship!, account, 'profile.more.block_domain')"
             />
             <CommonDropdownItem
+              is="button"
               v-else
               :text="$t('menu.unblock_domain', [getServerName(account)])"
               icon="i-ri:restart-line"
               :command="command"
-              @click="toggleBlockDomain(relationship!, account)"
+              @click="toggleBlockDomain(relationship!, account, 'profile.more.unblock_domain')"
             />
           </template>
 
           <CommonDropdownItem
+            is="button"
             :text="$t('menu.report_account', [`@${account.acct}`])"
             icon="i-ri:flag-2-line"
             :command="command"
-            @click="openReportDialog(account)"
+            @click="report()"
           />
         </template>
 
         <template v-else>
           <NuxtLink to="/pinned">
-            <CommonDropdownItem :text="$t('account.pinned')" icon="i-ri:pushpin-line" :command="command" />
+            <CommonDropdownItem is="button" :text="$t('account.pinned')" icon="i-ri:pushpin-line" :command="command" data-glean="profile.more.goto_pinned" />
```

Comment:
This until the end of the file are more links lacking 'target account'-identifiers